server/glide.yaml

```diff
@@ -1,2 +1,9 @@
 package: github.com/8tomat8/SSU-Golang-252-Chat/server
-import: []
+import:
+- package: github.com/gorilla/mux
+  version: ~1.4.0
+- package: github.com/Sirupsen/logrus
+  version: ~1.0.2
+- package: github.com/8tomat8/SSU-Golang-252-Chat
```

Comment:
Your internal packages should not be listed here. Otherwise you will have issues with adding changes